test/database/int.spec.ts

```diff
@@ -2809,6 +2809,16 @@ describe('database', () => {
     }
   })
 
+  it('should update simple', async () => {
+    const post = await payload.create({ collection: 'posts', data: { title: 'hello' } })
+    const res = await payload.db.updateOne({
+      where: { id: { equals: post.id } },
+      data: { title: 'hello updated' },
+      collection: 'posts',
+    })
+    expect(res.title).toBe('hello updated')
+  })
+
   it('should support x3 nesting blocks', async () => {
```

Comment:
I think some tests for groups and tabs would be good as well since they use optimized db calls as well